pkg/project/auth/reviewer.go

```diff
@@ -0,0 +1,66 @@
+package auth
+
+import (
+	"fmt"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+)
+
+// Review is a list of users and groups that can access a resource
+type Review interface {
```

Comment:
Why not return the RAR request itself here?  Would like to hear pro/con, I hate getting an object (that means roughly the same thing as what we need) and then throwing it away and creating a new object without a good justification.